bigtable/tableadmin/tableadmin.py

```diff
@@ -0,0 +1,253 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Demonstrates how to connect to Cloud Bigtable and run some basic operations.
+#     http://www.apache.org/licenses/LICENSE-2.0
+Prerequisites:
+- Create a Cloud Bigtable cluster.
+  https://cloud.google.com/bigtable/docs/creating-cluster
+- Set your Google Application Default Credentials.
+  https://developers.google.com/identity/protocols/application-default-credentials
+
+Operations performed:
+- Create bigtable.
+- List bigtables existing in current project, if any.
+- List metadata(Name) of created bigtable.
+- Create Column Families with different GC rules.
+    - GC Rules like: MaxAge, MaxVersions, Union, Intersection and Nested.
+- Delete bigtable.
+"""
+
+import argparse
+import datetime
+
+from google.cloud import bigtable
+
+
+def run_table_operations(project_id, instance_id, table_id):
+    ''' Create bigtable and perform different operations on it.
+
+    :type project_id: str
+    :param project_id: Project id of the client.
+
+    :type instance_id: str
+    :param instance_id: Instance of the client.
+
+    :type table_id: str
+    :param table_id: Table id to create table.
+    '''
+
+    client = bigtable.Client(project=project_id, admin=True)
+    instance = client.instance(instance_id)
+    table = instance.table(table_id)
+
+    # Check whether table exists in an instance.
+    # Create table if it does not exists.
+    print 'Checking if table {} exists...'.format(table_id)
+    if exists(instance, table_id):
+        print 'Table {} already exists.'.format(table_id)
+    else:
+        print 'Creating the {} table.'.format(table_id)
+        table.create()
+        print 'Created table {}.'.format(table_id)
+
+    tables = instance.list_tables()
+
+    # [START List existing tables in the instance.]
+    print 'Listing tables in current project...'
+    if tables != []:
+        for tbl in tables:
+            print tbl.name
+    else:
+        print 'No table exists in current project...'
+    # [END List existing tables in the instance.]
+
+    # Display name of the table.
+    print 'Printing table metadata...'
+    print table.name

```

Comment:
Is there no equivalent call to the `table.getMetadata` in Node.js ? @sduskis 
https://github.com/googleapis/nodejs-bigtable/blob/master/samples/tableadmin.js#L85